start/src/test/java/it/io/openliberty/guides/system/SystemIT.java

```diff
@@ -37,17 +39,19 @@
 @RunWith(Arquillian.class)
 public class SystemIT {
 
-    private final static String WARNAME = "arquillian-managed";
+    private final static String WARNAME = "arquillian-managed.war";
 
     // tag::deployment[]
     @Deployment(testable = true)
     public static WebArchive createSystemEndpointTestDeployment() {
-        WebArchive archive = ShrinkWrap.create(WebArchive.class, WARNAME + ".war")
+        WebArchive archive = ShrinkWrap.create(WebArchive.class, WARNAME)
```

Comment:
Also add package here instead of classes?

Reply:
Fixed